charts/policy-controller/values.schema.json

```diff
@@ -1,1281 +1,285 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "http://example.com/example.json",
+    "$schema": "http://json-schema.org/schema#",
```

Comment:
all those remove are correct?

Reply:
i used the `helm schema-gen` plugin, as discussed in #https://github.com/sigstore/helm-charts/issues/192. I assumed it is a valid operation.